### PR TITLE
 MPT-43 Increase wait time in DeploymentValidation script 

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
:green_apple: 

test this please

**Comment by chibenwa** [comment]
:green_apple: again

test this please

**Comment by chibenwa** [comment]
:green_apple: again

test this please

File: dockerfiles/compilation/java-8/compile.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ git checkout $SHA1
 if [ "$SKIPTESTS" = "skipTests" ]; then
    mvn package -DskipTests ${MVN_ADDITIONAL_ARG_LINE}
 else
-   mvn package ${MVN_ADDITIONAL_ARG_LINE}
+   mvn package -DskipTests ${MVN_ADDITIONAL_ARG_LINE}
```
Comment (trantienduchn) [inline dockerfiles/compilation/java-8/compile.sh]: this command it looks same to the above command, then `$SKIPTESTS` variable become useless?

Comment (Arsnael) [reply dockerfiles/compilation/java-8/compile.sh]: That was to forcefully skip the tests and test just the deployment

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**